requirements.txt

```diff
@@ -9,8 +9,9 @@
 
 # Direct dependencies (except pip, setuptools, wheel):
 
-# zhmcclient @ git+https://github.com/zhmcclient/python-zhmcclient.git@master
-zhmcclient>=1.12.1
+# TODO-ZHMC: Enable again once 1.13.4 is released
+zhmcclient @ git+https://github.com/zhmcclient/python-zhmcclient.git@stable_1.13
+# zhmcclient>=1.13.4
```

Comment:
same, you completely drop the coreq?

Reply:
The syntax on line 13 is the replacement until 1.13.4 is released. It defines the dependency to a package named zhmcclient that can be obtained at that link.

That is the temporary way to get the zhmcclient fix that is needed, before the next version of zhmcclient is released. I have created issue https://github.com/zhmcclient/zhmccli/issues/566 to get back to the released version of zhmcclient, before we release zhmccli.